app/lib/search/backend.dart

```diff
@@ -139,6 +140,16 @@ class SearchBackend {
     return emails.toList()..sort();
   }
 
+  Future<List<String>> _buildUploaderEmails(Package p) async {
+    if (p.publisherId == null) {
```

Comment:
```suggestion
    if (p.publisherId != null) {
      return <String>[];
    }
```
nit: prefer to return early...